internal/serve/httphandler/payments_handler.go

```diff
@@ -32,3 +32,20 @@ func (h PaymentsHandler) SubscribeAddress(w http.ResponseWriter, r *http.Request
 		return
 	}
 }
+
+func (h PaymentsHandler) UnsubscribeAddress(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	var reqBody PaymentsSubscribeRequest
+	err := httpdecode.DecodeJSON(r, &reqBody)
+	if err != nil {
+		httperror.BadRequest("Invalid request body").Render(w)
+		return
+	}
+
+	err = h.Models.Payments.UnsubscribeAddress(ctx, reqBody.Address)
+	if err != nil {
```

Comment:
Can we log and send a report to the sentry when a 5xx happens?

Reply:
Applied the same resolution as in the `/subscribe` PR: https://github.com/stellar/wallet-backend/pull/6#discussion_r1593921662

Reply:
Thank you.